src/schemas/json/bacalhau-job.json

```
@@ -0,0 +1,506 @@
+{
+  "$defs": {
+    "Deal": {
+      "properties": {
+        "Concurrency": {
+          "type": "integer"
+        },
+        "Confidence": {
+          "type": "integer"
+        },
+        "MinBids": {
+          "type": "integer"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object"
```

```suggestion
      "type": "object",
      "title": "title"
```

Optional fix: `title` can be added to enhance VS Code editing experience.